Gem/Transactions/Scenes/TransactionsScene.swift

```diff
@@ -48,15 +43,35 @@ struct TransactionsScene: View {
             )
         }
         .onAppear {
-            Task {
-                await model.fetch()
-            }
+            onAppear()
         }
     }
 }
 
-struct TransactionsScene_Previews: PreviewProvider {
-    static var previews: some View {
-        TransactionsScene(model: .init(wallet: .main, type: .all, service: .main))
+// MARK: - Actions
+
+extension TransactionsScene {
+    private func onAppear() {
+        refreshAcitities()
+    }
+
+    private func onDataRefresh() {
+        refreshAcitities()
     }
 }
+
+// MARK: - Effects
+
+extension TransactionsScene {
+    private func refreshAcitities() {
```

Comment:
```suggestion
    private func fetch() {
```

let's use `fetch` as main method to fetch data for the scene